media/public/include/IWebAudioPlayer.h

```diff
@@ -137,7 +137,7 @@ class IWebAudioPlayer
      *
      * @retval true on success.
      */
-    virtual bool getBufferAvailable(uint32_t &availableFrames, std::shared_ptr<WebAudioShmInfo> &webAudioShmInfo) = 0;
+    virtual bool getBufferAvailable(uintptr_t &availableFrames, std::shared_ptr<WebAudioShmInfo> &webAudioShmInfo) = 0;
```

Comment:
Can you explain why you change uint32_t to uintptr_t?

Reply:
The change is done to uintptr_t, as 64 bit build doesn't support uint32_t .